frontend/src/modules/Dashboards/views/DashboardImportForm.js

```diff
@@ -136,7 +81,7 @@ const DashboardImportForm = (props) => {
             dashboardId: values.dashboardId,
             environmentUri: values.environment.environmentUri,
             description: values.description,
-            SamlGroupName: values.SamlGroupName,
+            SamlAdminGroupName: values.SamlAdminGroupName,
```

Comment:
Can we keep the key value `SamlGroupName` ? Otherwise - wouldn't we also have to change the Backend importDashboard input type and resolver logic?